arrow-site/docs/docs/quickstart/setup/README.md

```diff
@@ -30,9 +28,6 @@ In your project's root `build.gradle`, append these repositories to your list:
 allprojects {
     repositories {
         mavenCentral()
-        jcenter()
-        maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" }
-        maven { url "https://oss.jfrog.org/artifactory/oss-snapshot-local/" } // for SNAPSHOT builds
```

Comment:
OSS is still being used for SNAPSHOTs

Reply:
I thought that was also going away for some reason.

Reply:
Added the snapshots repo back